x-pack/plugins/actions/server/routes/list_action_types.ts

```diff
@@ -6,28 +6,37 @@
  */
```

Comment:
Should this file be renamed `connector_types` to reflect the updated route name?

Reply:
Yes!